index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Supreeti Portfolio</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+        <div class="logo">
+          <a href="#"
+          ><img src="images/My Logo.png" alt=""
+        /></a> </div>
+        <nav>
+         <div class="menu-icon"> 
+            <img src="images/Item Navigation.png" alt=""/>
+         </div>
```

Comment:
As requested by th preious reviewer, please follow the figam template and use sematic HTML to make develop your portfolio , this link is supposed to be a hamburger icon, see the figma template :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200120214-6842c7a0-0dee-4de8-b5b5-131fda953faa.png)
you design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200120190-54beef4a-945e-44e9-b040-963e3a628100.png)